src/clj/puppetlabs/trapperkeeper/services/watcher/filesystem_watch_service.clj

```diff
@@ -12,7 +12,7 @@
 
   (init
     [this context]
-    {:watchers (atom [])})
+    (assoc context :watchers (atom [])))
```

Comment:
It wasn't failing a test... but it seemed that unless we could guarantee tk-fs-watcher was always initialized first we would at some point swallow someone's context.

FWIW, w came across this because we did have a test that produced error messages when we initially promoted tk-fs-watcher 1.0 into pe-file-sync. It was ensuring that the storage service shutdown during its init phase if passed an invalid config. After a while we realized that with tk-fs-watcher no longer having a dependency on tk-scheduler the storage-service no longer needed it either, and the test was failing because  the scheduler wasn't getting to its init phase during the test, but was running its stop phase, failing to deref an atom it expected to exist in its context that it would have put in there during its init. That led us to search for things-that-could-be-trashing-a-services-context-during-init and we found this.